### PR TITLE
add ebook and course on th resource

### PR DESCRIPTION
**Pull request by minatorak**
## What does this PR do?
Add resource

### How do we know it's really free?
for golang https://www.golang-book.com/public/pdf/gobook.0.pdf
for python have copyright free for education but do not sell
for all course available on owner youtube channel

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

### CODE REVIEW

File: books/free-programming-books-th.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 ### Go
 
 * [ภาษา Go ตอน 1 ติดตั้ง และ Run Hello World](https://medium.com/odds-team/%E0%B8%AA%E0%B8%A3%E0%B8%B8%E0%B8%9B%E0%B8%81%E0%B8%B2%E0%B8%A3%E0%B9%80%E0%B8%A3%E0%B8%B5%E0%B8%A2%E0%B8%99%E0%B8%9E%E0%B8%B7%E0%B9%89%E0%B8%99%E0%B8%90%E0%B8%B2%E0%B8%99%E0%B8%A0%E0%B8%B2%E0%B8%A9%E0%B8%B2-go-%E0%B9%81%E0%B8%9A%E0%B8%9A-step-by-step-%E0%B8%88%E0%B8%B2%E0%B8%81-course-pre-ultimate-go-by-p-yod-%E0%B8%95%E0%B8%AD%E0%B8%99-1-%E0%B8%95%E0%B8%B4%E0%B8%94%E0%B8%95%E0%B8%B1%E0%B9%89%E0%B8%87-%E0%B9%81%E0%B8%A5%E0%B8%B0-d9ac7913e9a4) - Chaiyarin Niamsuwan
+* [An Introduction to programming in Go](https://www.dropbox.com/sh/is3hwdqa1dpsb99/AACb9QvxPEUo1Z-tD0Nz2KNZa?dl=0) - Caleb Doxsey
```
Comment (shrn01) [inline books/free-programming-books-th.md:25]: Dropbox links are not allowed. Please provide official links to the resources

File: books/free-programming-books-th.md [hunk 2]
```diff
@@ -44,4 +45,6 @@
 
 ### Python
 
+* [Programming Expert with Python](https://drive.google.com/file/d/0B6N6HcO5p2-9ck50d3d4bG9MRnc/view?usp=sharing&resourcekey=0-T1LaEEp7tQANUSx7iNYgpA) -  ผศ.ดร.สุชาติ คุ้มมะณี
```
Comment (shrn01) [inline books/free-programming-books-th.md:48]: Drive links are not allowed. Please provide official links to the resources

File: books/free-programming-books-th.md [hunk 3]
```diff
@@ -44,4 +45,6 @@
...
 * [Python ๑๐๑](https://www.cp.eng.chula.ac.th/books/python101/) - กิตติภณ พละการ, กิตติภพ พละการ, สมชาย ประสิทธิ์จูตระกูล, สุกรี สินธุภิญโญ
+* [Python Programming for Networking and Security](https://drive.google.com/drive/folders/1zRR4KAS0H_Kn5_PVYTIGCRPEALO6R2SO?fbclid=IwAR2RBY1d-3_q1_7YzEPZgjRlInParIrIu6uQfXQ2cwmI2HIsKIP7grj2F5k) - ผศ.ดร.สุชาติ คุ้มมะณี
```
Comment (shrn01) [inline books/free-programming-books-th.md:50]: Drive links are not allowed. Please provide official links to the resources

File: courses/free-courses-th.md [hunk 4]
```diff
@@ -30,6 +44,11 @@
 * [ภาษา C++](http://marcuscode.com/lang/cpp) - MarcusCode
 
 
+### Databases
+
+* [Database Systems](https://youtube.com/playlist?list=PL7fB6_3v0nhxRBOP44PL8SnhXnauinlb2) - Aj Earn KMUTNB
```
Comment (shrn01) [inline courses/free-courses-th.md:49]:
```suggestion
* [วิชา Database System (ปี 58/1)](https://youtube.com/playlist?list=PL7fB6_3v0nhxRBOP44PL8SnhXnauinlb2) - Aj Earn KMUTNB
```
Dont change the resource name

File: courses/free-courses-th.md [hunk 5]
```diff
@@ -15,6 +18,17 @@
...
 
+### Algorithms & Data Structures
+
+* [การออกแบบและวิเคราะห์อัลกอริทึม](https://youtube.com/playlist?list=PL0ROnaCzUGB65_YkASLAEmcW_mtxFtq4m) - สมชาย ประสิทธิ์จูตระกูล
+
+
+### Artificial Intelligence
+
+* [Natural language processing](https://youtube.com/playlist?list=PLcBOyD1N1T-PIYnPZ9_iHtug9e-BcHIob) - EkapolC
+* [Pattern Recognition](https://youtube.com/playlist?list=PLcBOyD1N1T-MnWcKQZqE8FXrgoiiVdXvI) - EkapolC
```
Comment (shrn01) [inline courses/free-courses-th.md:29]:
```suggestion
* [Pattern 2022](https://youtube.com/playlist?list=PLcBOyD1N1T-MnWcKQZqE8FXrgoiiVdXvI) - EkapolC
```
Dont change the resource name

File: courses/free-courses-th.md [hunk 6]
```diff
@@ -15,6 +18,17 @@
...
 
 
+### Algorithms & Data Structures
+
+* [การออกแบบและวิเคราะห์อัลกอริทึม](https://youtube.com/playlist?list=PL0ROnaCzUGB65_YkASLAEmcW_mtxFtq4m) - สมชาย ประสิทธิ์จูตระกูล
+
+
+### Artificial Intelligence
+
+* [Natural language processing](https://youtube.com/playlist?list=PLcBOyD1N1T-PIYnPZ9_iHtug9e-BcHIob) - EkapolC
```
Comment (shrn01) [inline courses/free-courses-th.md:28]:
```suggestion
* [NLP 2021](https://youtube.com/playlist?list=PLcBOyD1N1T-PIYnPZ9_iHtug9e-BcHIob) - EkapolC
```
Dont change the resource name

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: books/free-programming-books-th.md (hunk 1 above)
Comment (minatorak) [reply books/free-programming-books-th.md:25]: agree, I'm will talk to writer
thank suggestion

File: books/free-programming-books-th.md [hunk 7]
```diff
@@ -46,3 +46,4 @@
 ### Python
 
 * [Python ๑๐๑](https://www.cp.eng.chula.ac.th/books/python101/) - กิตติภณ พละการ, กิตติภพ พละการ, สมชาย ประสิทธิ์จูตระกูล, สุกรี สินธุภิญโญ
+* [Python Programming for Networking and Security](https://www.mebmarket.com/ebook-216597-%E0%B8%81%E0%B8%B2%E0%B8%A3%E0%B9%80%E0%B8%82%E0%B8%B5%E0%B8%A2%E0%B8%99%E0%B9%82%E0%B8%9B%E0%B8%A3%E0%B9%81%E0%B8%81%E0%B8%A3%E0%B8%A1%E0%B9%84%E0%B8%9E%E0%B8%98%E0%B8%AD%E0%B8%99%E0%B8%AA%E0%B8%B3%E0%B8%AB%E0%B8%A3%E0%B8%B1%E0%B8%9A%E0%B8%A3%E0%B8%B0%E0%B8%9A%E0%B8%9A%E0%B9%80%E0%B8%84%E0%B8%A3%E0%B8%B7%E0%B8%AD%E0%B8%82%E0%B9%88%E0%B8%B2%E0%B8%A2%E0%B9%81%E0%B8%A5%E0%B8%B0%E0%B8%84%E0%B8%A7%E0%B8%B2%E0%B8%A1%E0%B8%9B%E0%B8%A5%E0%B8%AD%E0%B8%94%E0%B8%A0%E0%B8%B1%E0%B8%A2-Python-Programming-for-Networking-and-Security?fbclid=IwAR31YnSXlslGcjCEvwdhSEf8tI1LXm9e9bjwVNnb6F2kHYGAynFoUHoLxeU) - ผศ.ดร.สุชาติ คุ้มมะณี
```
Comment (eshellman) [inline books/free-programming-books-th.md:49]:
```suggestion
```
we don't list email-required books

**Pull request merged**